src/widget/widget.h

```diff
@@ -181,6 +183,9 @@ private slots:
     void onSplitterMoved(int pos, int index);
     void processOfflineMsgs();
     void friendListContextMenu(const QPoint &pos);
+    void onDesktopNotificationsToggled(NotificationBackend *notificationBackend);
+    void notifyAvInvite(int friendId);
```

Comment:
1. Please rename to `onNotifyAvInvite`…
2. Same as above -> change type to `uint32_t` or better `quint32`